misc/show-installed-tools/env0.yml

```diff
@@ -4,6 +4,7 @@ deploy:
   steps:
```

Comment:
I'm going to use this template to check that we have all of the permissions we need, inside a test in:
`e2e/template-integration/saas`

Should I rename this file to `show-and-ensure-permissions-for-installed-tools/env0.yml`?

Reply:
@sabasabo whatever you think